src/main/resources/templates/findings/sarif.peb

```diff
@@ -0,0 +1,54 @@
+{
+  "version": "2.1.0",
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0.json",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "OWASP Dependency-Track",
+          "version": "{{ dependencyTrackVersion }}",
+          "informationUri": "https://dependencytrack.org/",
+          "rules": [{% for finding in findings %}
+            {
+              "id": "{{ finding.vulnerability.vulnId }}",
+              "name": "{{ finding.component.name }} - {{ finding.vulnerability.cweName }}",
+              "shortDescription": {
+                "text": "{{ finding.vulnerability.description | split('\n') | join(' ') | trim }}"
```

Comment:
For free-text values, consider using `| escape(strategy="json")` to prevent the SARIF JSON from breaking, for example when the `description` includes double quotes.

You can also set the default escaping strategy when you construct the template engine, e.g.:

https://github.com/DependencyTrack/dependency-track/blob/3b52284f98526f533bd8b7b705e707f6f5847102/src/main/java/org/dependencytrack/notification/publisher/WebhookPublisher.java#L28

Reply:
Thank you!